app.js

```diff
@@ -0,0 +1,23 @@
+const express = require("express");
+
+const { getTopics ,
+      } = require("./controllers/topicsController");
+
+const app = express();
+app.use(express.json());
+
+app.get("/api/topics", getTopics);
+
+app.use((err, req, res, next) => {
```

Comment:
you haven't added any tests that check for errors yet so you dont need this code yet